### PR TITLE
REF: Update broker to new FS API.

### PR DESCRIPTION
**Pull request by danielballan**
Do not merge. This might be fine, but it was a hasty fix to make the docs build and might probably fail in tests.

### CODE REVIEW

**Comment by danielballan** [comment]
Ready to merge when Travis concurs.

**Comment by danielballan** [comment]
New feature added to this PR: when the broker is imported, it inspects `filestore/handlers.py` for Handlers that have a `specs` attribute, and it registers them accordingly. This could and should be done is a smarter way, but it works for now.

File: dataportal/broker/handler_registration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+def register_builtin_handlers():
+    "Register all the handlers built in to filestore."
+    # TODO This will blow up if any non-leaves in the class heirarchy
+    # have non-empty specs. Make this smart later.
+    for cls in vars(handlers).values():
+        if isinstance(cls, type) and issubclass(cls, HandlerBase):
```
Comment (tacaswell) [inline dataportal/broker/handler_registration.py:15]: If you are going to filter on subclass of `HandlerBase` wight as well ask it what it's sub-classes are.

Comment (danielballan) [inline dataportal/broker/handler_registration.py:15]: Ha. Just as we do in pims. Stand by.

Comment (danielballan) [inline dataportal/broker/handler_registration.py:15]: Actually, maybe not. What if a user has a couple handlers already defined in the namespace, but he doesn't want to register them?

Comment (danielballan) [inline dataportal/broker/handler_registration.py:15]: To put it another way, PIMS looks for all subclasses of FramesSequence each time `open()` is called, and so it is always up to date with the complete list of subclasses, not only those defined at import time.

Comment (tacaswell) [inline dataportal/broker/handler_registration.py:15]: These are the some-what dicey issues I was worried about with auto-registering ;)

**Pull request merged**